### ISSUE TITLE
Plan to release training code？

### ISSUE THREAD

**Issue by huangjch526** [FoundationVision/FlashVideo#3]

**Comment by jshilong** [comment]
Considering the high requirements for computing resources and data works, and our inability to release the data, I initially assumed there would be limited interest in the training code. However, if there is significant demand, I can certainly arrange it for release - it should not pose a major issue.

If you are affiliated with an organization working on a video foundation model and wish to scale up a similar cascade framework, please feel free to get in touch with me. You are most welcome.

**Comment by linjing7** [comment]
Hi, I'm also interested in the training code. It would be really helpful if you could release it  :)

**Comment by jshilong** [comment]
I am currently engaged in other projects and due to limited capacity, it may take some time to organize the code. The main concern is that the training code depends on the augmentation from the [OpenMMLab project](https://github.com/open-mmlab/mmagic). I must extract the relevant code to prevent you from undergoing a complex installation process. I will create a TODO to remind myself about this task. Please stay tuned.

**Comment by eisneim** [comment]
you can relaase just the training part first and dataset format example, the hard part is  the loss function;

**Comment by jshilong** [comment]
@eisneim @linjing7 @huangjch526 

I include code related to the training loss function in this PR. I plan to add dataset examples and configuration details soon. It has been a particularly busy time for me recently, and I hope for your understanding.

https://github.com/FoundationVision/FlashVideo/pull/12
